plugins/GSdx/Renderers/Common/GSOsdManager.h

```diff
@@ -1,98 +1,39 @@
-/*
- *	Copyright (C) 2014-2016 Gregory hainaut
- *	Copyright (C) 2016-2016 Jason Brown
- *
- *  This Program is free software; you can redistribute it and/or modify
- *  it under the terms of the GNU General Public License as published by
- *  the Free Software Foundation; either version 2, or (at your option)
- *  any later version.
- *
- *  This Program is distributed in the hope that it will be useful,
- *  but WITHOUT ANY WARRANTY; without even the implied warranty of
- *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- *  GNU General Public License for more details.
- *
- *  You should have received a copy of the GNU General Public License
- *  along with GNU Make; see the file COPYING.  If not, write to
- *  the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA USA.
- *  http://www.gnu.org/copyleft/gpl.html
- *
- */
-
```

Comment:
My concern this this code has moved from OsdManager to OsdManagerOGL. I think the code should stay with the author.

That said, if you think otherwise I'll restore it.

Reply:
Since the code was moved the copyright header author can be changed to.

` *	Copyright (C) 2018 PCSX2 Dev Team`